docs/guide/acquiring_data/database.rst

```diff
@@ -110,17 +110,17 @@ tags                   A list of :class:`tables.Tag` instances.
   ``observation_time_start``, ``observation_time_end``, ``instrument``,  ``size``,
   ``wavemin``, and ``wavemax`` are set by methods which use the VSO interface. In
   particular, these are :meth:`Database.add_from_vso_query_result`,
-  :meth:`Database.download` and   possibly :meth:`Database.fetch`.
+  ``Database.download`` and   possibly :meth:`Database.fetch`.
```

Comment:
Is this not a method anymore?

Reply:
Doesn't look like it, possibly it was never a method?

Reply:
Probably best to remove it then.